README.md

```diff
@@ -2,11 +2,72 @@
 A repository for deploying OpenShift clusters with Red Hat Advanced Cluster Management for Kubernetes and OpenShift GitOps.  
 All cluster lifecycle is managed by Argo CD, including the Argo configuration itself.
 
+## Prerequisites
+In order to successfully use the contents of this repository, you need to have an OpenShift cluster deployed *with* storage available and ready to consume.  The methods of deployment of this OpenShift cluster can be:
+- Red Hat ACM
+- [Baremetal Installer Provisioned Infrastructure] (https://docs.openshift.com/container-platform/4.12/installing/installing_bare_metal_ipi/ipi-install-overview.html)
+- [Crucible Ansible Playbooks] (https://github.com/redhat-partner-solutions/crucible)
+- [Agent Based Installer] (https://docs.openshift.com/container-platform/4.12/installing/installing_with_agent_based_installer/preparing-to-install-with-agent-based-installer.html)
```

Comment:
I think we already solved this linking issue, probably need to rebase with main or resolve conflicts